.github/workflows/run-tests.yml

```diff
@@ -152,4 +152,14 @@ jobs:
     uses: ./.github/workflows/run-mill-action.yml
     with:
       java-version: '11'
-      buildcmd: ./mill -i mill.scalalib.scalafmt.ScalafmtModule/checkFormatAll __.sources; ./mill -i __.mimaReportBinaryIssues; ./mill -i __.fix --check
+      buildcmd: |
```

Comment:
Rather than using Spotless it in the main Mill repo and CI jobs, let's put it in an example test for now in each of the three languages `example/{language}lib/linting/` folders and include it in the respective `linting.adoc` on the docsite